trendmicro_deepsecurity/icon_trendmicro_deepsecurity/actions/deploy_rules/action.py

```diff
@@ -0,0 +1,75 @@
+import komand
+from .schema import DeployRulesInput, DeployRulesOutput, Input, Output, Component
+# Custom imports below
+
+import requests
+import json
+
+from icon_trendmicro_deepsecurity.util.shared import tryJSON
+from icon_trendmicro_deepsecurity.util.shared import checkResponse
+
+class DeployRules(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='deploy_rules',
+                description=Component.DESCRIPTION,
+                input=DeployRulesInput(),
+                output=DeployRulesOutput())
+
+    def run(self, params={}):
+        """
+        Set IPS rules in Deep Security
+        """
+
+        # Get parameters
+        self.computer_or_policy=params.get("computer_or_policy")
```

Comment:
for parameters we like to use the `self.computer_or_policy = params.get(Input.COMPUTER_OR_POLICY)`

By using `Input` you prevent miss-typing from messing things up.
for example if you put params.get("computer-or_policy") The plugin would "work" your  variable would just equal None

Reply:
done